containers/java-postgres/README.md

```diff
@@ -0,0 +1,132 @@
+
+```yaml
+args:
+  VARIANT: 11
+  NODE_VERSION: "10" # Set to "none" to skip Node.js installation
+```
+
+### Adding the definition to your folder
+
+1. If this is your first time using a development container, please see getting started information on [setting up](https://aka.ms/vscode-remote/containers/getting-started) Remote-Containers or [creating a codespace](https://aka.ms/ghcs-open-codespace) using GitHub Codespaces.
+
+2. Start VS Code and open your project folder or connect to a codespace.
+
+3. Press <kbd>F1</kbd> select and **Add Development Container Configuration Files...** command for **Remote-Containers** or **Codespaces**.
+
+   > **Note:** If needed, you can drag-and-drop the `.devcontainer` folder from this sub-folder in a locally cloned copy of this repository into the VS Code file explorer instead of using the command.
+
+4. Select this definition. You may also need to select **Show All Definitions...** for it to appear.
+
+5. Finally, press <kbd>F1</kbd> and run **Remote-Containers: Reopen Folder in Container** or **Codespaces: Rebuild Container** to start using the definition.
+
+## Testing the definition
+
+This definition includes some test code that will help you verify it is working as expected on your system. Follow these steps:
+
+1. If this is your first time using a development container, please follow the [getting started steps](https://aka.ms/vscode-remote/containers/getting-started) to set up your machine.
+2. Clone this repository.
+3. Start VS Code, press <kbd>F1</kbd>, and select **Remote-Containers: Open Folder in Container...**
+4. Select the `containers/java-postgres` folder.
+5. After the folder has opened in the container, press <kbd>F5</kbd> to start the project.
+6. You should see "Hello Remote World!" in the a Debug Console after the program executes.
+7. From here, you can add breakpoints or edit the contents of the `test-project` folder to do further testing.
+
+## Testing the PostgreSQL container
+
+The `docker-compose` file sets up a networked PostgreSQL database that is accessible from the Java Dev Container. The port is forwarded to `localhost:5432` by default, but can be changed in the [devcontainer.json](.devcontainer\devcontainer.json).
+
+1. After starting the Dev Container as above, you can use the **Debug Console** to test the PostgreSQL Database.
+2. The [AppTest.java](test-project\src\test\java\com\mycompany\app\AppTest.java) contains a Test Method, `testIP` which will ping the Postgres Database using it's default container name, `postgresdb`.
+3. Running this test will let you know that the PostgreSQL DB is accessible. **This does not make or authorize a connection to the database, only checks for connectivity between containers.**
+4. Alternatively, running `./test.sh` will also run all the connectivity tests and verify that the PostgresDB is actually accessible. 
```

Comment:
I was able to run the test.sh bash script, but I had to find it first (under the test-project folder).

Reply:
Ah, let me add a link to it. I've got links to other things, so I might as well do it here too.

Reply:
Done!